include/materials/texture.h

```diff
@@ -60,4 +60,21 @@ class image_texture : public texture {
     image image_data;
 };
 
+struct RGBA {
+  float A = 1.0;
+  color RGB = color(0,1,1);
+};
+// Specifically takes in PNGs and also accounts for alpha channel.
+class pixel_image_texture : public texture {
```

Comment:
classes should be renamed with camel case with first char capitalized
PixelImageTexture